shared/src/sector/registered_proof.rs

```diff
@@ -92,14 +98,19 @@ impl RegisteredSealProof {
             | StackedDRG8MiBV1P1
             | StackedDRG2KiBV1P1_Feat_SyntheticPoRep
             | StackedDRG512MiBV1P1_Feat_SyntheticPoRep
-            | StackedDRG8MiBV1P1_Feat_SyntheticPoRep => Ok(192),
+            | StackedDRG8MiBV1P1_Feat_SyntheticPoRep
+            | StackedDRG2KiBV1P1_Feat_NiPoRep
+            | StackedDRG512MiBV1P1_Feat_NiPoRep
+            | StackedDRG8MiBV1P1_Feat_NiPoRep => Ok(192),
 
             StackedDRG32GiBV1
             | StackedDRG64GiBV1
             | StackedDRG32GiBV1P1
             | StackedDRG64GiBV1P1
             | StackedDRG32GiBV1P1_Feat_SyntheticPoRep
-            | StackedDRG64GiBV1P1_Feat_SyntheticPoRep => Ok(1920),
+            | StackedDRG64GiBV1P1_Feat_SyntheticPoRep
+            | StackedDRG32GiBV1P1_Feat_NiPoRep
+            | StackedDRG64GiBV1P1_Feat_NiPoRep => Ok(1920),
```

Comment:
I'm not familiar with this size here, but is this value correct for NI-PoRep?

Reply:
Should it be 10x larger? @irene 

(We're never going to use this if we only support aggregation, but let's get it right).

Reply:
it should be 12.8x higher

Reply:
Can someone produce a proof and measure it? The last time I yoloed one of these values I halted proving on the network (testnet, luckily).

Reply:
tbh...not really yoloing here, but I agree producing a proof and measuring it would be the best way to know it.

Reply:
126 is the number  of circuits per NI-PoRep (thanks @vmx )
> https://github.com/filecoin-project/rust-fil-proofs/blob/0dfef1d8416c8240400fc188e90f0e55fa6b90ae/filecoin-proofs/src/constants.rs#L152

which means the expected size for NI PoRep proof is 192 * 10 * 12.6

As I said, having someone running a proof would be the ultimate confirmation

Reply:
We need the _exact_ size. I say yoloing because we're attempting to estimate the size by multiplying numbers. I'm assuming there's some rounding somewhere.

`ceil(192 * 10) * 12.6 != ceil(192 * 12.6) * 10`  (24192 versus 24200)

Reply:
I understand, but my take is we have 126 circuits, each circuit is 192 bytes. So you should not have issues with rounding no?
the 3 terms were to explain where we are coming from and the comparison with current porep.

ANyway, ofc let's run the proof if anyone can to it easily

Reply:
Got it, that makes sense.

Reply:
So given we have aggregation within the sector (i thought we did not want to consider aggregation at all here, but i was wrong), the correct value is 23092 bytes (thanks @rvagg for measuring it!).
[ Note that this matches what the Snarkpack paper says about aggregating Vs not aggregating 128 proofs , where they observe 24kb for the non aggregated case and 23kb for the aggregated case, ref here, section 7  https://eprint.iacr.org/2021/529.pdf ]

Reply:
going with `23_092` because it's what I measured from running it, 🤞